tests/vars.yml

```diff
@@ -1,14 +1,15 @@
 ---
 
-postgresql_version: 9.6
+postgresql_version: 10 
 
 postgresql_databases:
   - name: foobar
     owner: baz
 
 postgresql_users:
+
+  # postgresql 10 does not accepte unencrypted passwords 
```

Comment:
I would change the comment to:

`+  # PostgreSQL >= 10 does not accept unencrypted passwords`

Reply:
Done.